pkg/retry/retry.go

```diff
@@ -165,37 +165,36 @@ func (s *Server) handleRetry() http.Handler {
 						"guid", eventIdentifier.guid,
 						"error", err,
 					)
-					continue
-				}
-
-				// found an unaccepted error, check if its already in the events table
-				exists, err := s.datastore.DeliveryEventExists(ctx, s.eventsTableID, eventIdentifier.guid)
-				if err != nil {
-					logger.Errorw("failed to call BigQuery",
-						"method", "DeliveryEventExists",
+				} else {
```

Comment:
Should this just be `if !errors.As(err, &acceptedErr) {` insteaf of an if/else? Might clean up the look of it. IE only check if in table if we get an unknown error

Reply:
ack agreed